apps/core-app/src/main.tsx

```diff
@@ -1,19 +1,27 @@
+import { StrictMode } from 'react';
+import { createRoot } from 'react-dom/client';
```

Comment:
This applies to below comments to remove disable of eslint.
```suggestion
import ReactDOM from 'react-dom/client';
```